### PR TITLE
Add workflow_dispatch trigger to the default empty workflow

### PR DESCRIPTION
**Pull request by cschleiden**
This repository contains configuration for what users see when they click on the `Actions` tab and the setup page for Code Scanning.

It is not:
* A playground to try out scripts
* A place for you to create a workflow for your repository

---

**Please note that at this time we are only accepting new starter workflows for Code Scanning. Updates to existing starter workflows are fine.**

---

In the workflow and properties files:

- [x] The workflow filename of CI workflows should be the name of the language or platform, in lower case.  Special characters should be removed or replaced with words as appropriate (for example, "dotnet" instead of ".NET").

  The workflow filename of publishing workflows should be the name of the language or platform, in lower case, followed by "-publish".
- [x] Includes a matching `ci/properties/*.properties.json` file.
- [x] Use sentence case for the names of workflows and steps, for example "Run tests".
- [x] The name of CI workflows should only be the name of the language or platform: for example "Go" (not "Go CI" or "Go Build")
- [x] Include comments in the workflow for any parts that are not obvious or could use clarification.
- [x] CI workflows should run on `push` to `branches: [ $default-branch ]` and `pull_request` to `branches: [ $default-branch ]`.
- [x] Packaging workflows should run on `release` with `types: [ created ]`.
- [x] Code Scanning workflows should run on `push` to `branches: [ $default-branch, $protected-branches ]` and `pull_request` to `branches: [ $default-branch ]`. We also recommend a `schedule` trigger of `cron: $cron-weekly`.

Some general notes:

- [x] This workflow must only use actions that are produced by GitHub, [in the `actions` organization](https://github.com/actions), **or**
- [x] This workflow must only use actions that are produced by the language or ecosystem that the workflow supports.  These actions must be [published to the GitHub Marketplace](https://github.com/marketplace?type=actions).  We recommend that these actions be referenced using the full 40 character hash of the action's commit instead of a tag.  Additionally, workflows must include the following comment at the top of the workflow file:
    ```
    # This workflow uses actions that are not certified by GitHub.
    # They are provided by a third-party and are governed by
    # separate terms of service, privacy policy, and support
    # documentation.
    ```
- [x] Automation and CI workflows should not send data to any 3rd party service except for the purposes of installing dependencies.
- [x] Automation and CI workflows cannot be dependent on a paid service or product.

### CODE REVIEW

File: ci/blank.yml [hunk 1]
```diff
@@ -2,14 +2,17 @@
 
 name: CI
 
-# Controls when the action will run. Triggers the workflow on push or pull request
-# events but only for the $default-branch branch
+# Controls when the action will run. 
 on:
+  # Triggers the workflow on push or pull request events but only for the $default-branch branch
```
Comment (mscoutermarsh) [inline ci/blank.yml:7]:
```suggestion
  # Triggers the workflow on push or pull request events but only for the default branch
```

Maybe? Reads a little easier.

**Review by mscoutermarsh — APPROVED** [review]

File: ci/blank.yml (hunk 1 above)
Comment (cschleiden) [reply ci/blank.yml:7]: @mscoutermarsh `$default-branch` is automatically replaced with the name of the actual default branch when you use this workflow

File: ci/blank.yml [hunk 2]
```diff
@@ -2,14 +2,17 @@
...
   push:
     branches: [ $default-branch ]
   pull_request:
     branches: [ $default-branch ]
 
+  # This allows you to run this workflow manually from the Actions tab
```
Comment (cschleiden) [inline ci/blank.yml:13]: Maybe just? 
```suggestion
  # Allows you to run this workflow manually from the Actions tab
```

File: ci/blank.yml (hunk 1 above)
Comment (mscoutermarsh) [reply ci/blank.yml:7]: Yeah I know, I read it in my head as "default branch default branch"

**Review by andymckay — APPROVED** [review]
Looks good, let's do it 🚢 👍🏾

**Pull request merged**